src/sentry/receivers/onboarding.py

```diff
@@ -20,6 +24,23 @@
 from sentry.utils.javascript import has_sourcemap
 
 
+def check_for_onboarding_complete(organization):
```

Comment:
Probably not a huge deal, but I think it'd be worth caching this value.

This `check_for_onboarding_complete` is called on every single event, and in lots of cases, this will be 2 queries. At least 1 to check for `exists()`.